app/src/main/java/helium314/keyboard/keyboard/internal/keyboard_parser/floris/KeyCode.kt

```diff
@@ -132,8 +130,10 @@ object KeyCode {
     const val SHIFT_ENTER =               -10005
     const val ACTION_NEXT =               -10006
     const val ACTION_PREVIOUS =           -10007
+    const val MOVE_PAGE_UP =              -10008
+    const val MOVE_PAGE_DOWN =            -10009
     // Code value representing the code is not specified.
-    const val NOT_SPECIFIED =             -10008 // todo: not sure if there is need to have the "old" unspecified keyCode different, just test it and maybe merge
+    const val NOT_SPECIFIED =             -10010 // todo: not sure if there is need to have the "old" unspecified keyCode different, just test it and maybe merge
```

Comment:
Actually there is no need to have the `NOT_SPECIFIED` code last, you can also just leave it at `-10008`.
Though it would only be relevant if someone would use this code in a layout, which is quite unlikely.

Reply:
ok, I changed it back to the old code.
I also moved the newly added `CLIPBOARD_COPY_ALL` which I somehow missed to the app specific section